blocked-edges/4.19.0-InvalidArchitectureValueOfMachinesetsAnnotation.yaml

```diff
@@ -0,0 +1,9 @@
+to: 4.19.0
+from: 4[.]18[.].*
+url: https://issues.redhat.com/browse/MCO-1834
+name: InvalidArchitectureValueOfMachinesetsAnnotation
+message: |
+  Degrade machine-config cluster operator blocks the cluster update if a GCP or AWS cluster has machinesets with 
```

Comment:
> GCP or AWS cluster

If this is true then I'd prefer a GCP|AWS PromQL to avoid annoying unaffected releases

Reply:
Done.